__tests__/app.test.js

```diff
@@ -43,6 +43,31 @@ describe("/api", () => {
                 })
             })
         })
+        describe("/articles", () => {
+            test("returns array of articles with correct keys", () =>{
+                return request(app)
+                .get("/api/articles")
+                .expect(200)
+                .then(({body}) => {
+                    const { articles } = body
+                    expect(Object.keys(articles).length).toBeGreaterThan(0)
+                    articles.forEach((article) => {
+                        const ArtKeys = Object.keys(article)
+                        expect(ArtKeys).toEqual(expect.arrayContaining(["author","title","article_id","comment_count","topic","created_at","votes","article_img_url"]))
+                        expect(ArtKeys).not.toContain("body")
+                    });
```

Comment:
I'd check the types of values here too 

expect(typeof author).toBe('string') checks both the key and the value are there